WPFHospitalEditorUnitTests/RecommendAppointmentsTest.cs

```diff
@@ -0,0 +1,241 @@
+﻿using HealthcareBase.Model.HospitalResources;
+using HealthcareBase.Model.Schedule.Procedures;
+using HealthcareBase.Model.Schedule.SchedulingPreferences;
+using HealthcareBase.Model.Users.Employee.Doctors;
+using HealthcareBase.Model.Users.Generalities;
+using HealthcareBase.Model.Utilities;
+using HealthcareBase.Repository.ScheduleRepository.ProceduresRepository.Interface;
+using HealthcareBase.Repository.UsersRepository.EmployeesAndPatientsRepository.Interface;
+using HealthcareBase.Service.ScheduleService.ProcedureService;
+using HospitalWebApp.Dtos;
+using Moq;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using Xunit;
+
+namespace WPFHospitalEditorUnitTests
+{
+    public class RecommendAppointmentsTest
+    {
+        private readonly Mock<IShiftRepository> mockShiftRepository = new Mock<IShiftRepository>();
+        private readonly Mock<IExaminationRepository> mockExaminationRepository = new Mock<IExaminationRepository>();
+        private readonly Mock<IDoctorRepository> mockDoctorRepository = new Mock<IDoctorRepository>();
+        private readonly ExaminationService examinationService;
+        private static Dictionary<int, Doctor> doctors =  new Dictionary<int, Doctor>();
+        private static List<TimeInterval> timeIntervals =  new List<TimeInterval>();
+        private static Dictionary<int, Room> rooms = new Dictionary<int, Room>();
```

Comment:
There is no need for the variables to be static

Reply:
These variables must be static because they are used in Recommendation_request_data method which is also static.